Framework/Algorithms/test/GetEiTest.h

```diff
@@ -241,10 +240,10 @@ class GetEiTest : public CxxTest::TestSuite {
     for (int i = 0; i <= numBins; ++i) {
       const double xValue = 5.0 + 5.5 * i;
```

Comment:
Above, use `BinEdges xdata(numBins + 1, LinearGenerator(5.0, 5.5));` and get rid of this line and the assignment to `xdata` below.